ext/cuda/bilinearinterpolation.jl

```diff
@@ -0,0 +1,216 @@
+import ..ClimaInterpolations.BilinearInterpolation:
+    Bilinear,
+    set_source_range!,
+    interpolatebilinear!,
+    get_stencil_bilinear1d,
+    get_dims
+
+# CUDA limit per maximum number of logical blocks per grid dimension
+# The first dimension could have a higher limit, but a convervative
+# value is used here.
+_get_maxblocksperdim() = 65535
```

Comment:
Can you please open a new PR, pointing to documentation on whee this number comes from?